docker/mongodb-kubernetes-tests/tests/multicluster/multi_cluster_replica_set_scale_down.py

```diff
@@ -101,8 +105,8 @@ def test_ops_manager_has_been_updated_correctly_before_scaling():
 def test_scale_mongodb_multi(mongodb_multi: MongoDBMulti):
     mongodb_multi.load()
     mongodb_multi["spec"]["clusterSpecList"][0]["members"] = 1
-    mongodb_multi["spec"]["clusterSpecList"][1]["members"] = 1
-    mongodb_multi["spec"]["clusterSpecList"][2]["members"] = 1
+    mongodb_multi["spec"]["clusterSpecList"][1]["members"] = 0
```

Comment:
nit: let's add a small comment explaining why it's important here to have at least one cluster scaled down to 0